.github/workflows/cancel.yml

```diff
@@ -0,0 +1,21 @@
+name: Cancel Dispatch
+
+on:
+  repository_dispatch:
+    types: [cancel-command]
+jobs:
+  promote-release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set Env Variables
+      id: env_info
+      run: |
+        echo ::set-output name=issuetitle::$(jq -r .client_payload.github.payload.issue.title $GITHUB_EVENT_PATH)
+        echo ::set-output name=issuenm::$(jq .client_payload.github.payload.issue.number $GITHUB_EVENT_PATH)
+        echo ::set-output name=labels::$(jq .client_payload.github.payload.issue.labels $GITHUB_EVENT_PATH)
+        echo ::set-output name=requestor::$(jq .client_payload.github.actor $GITHUB_EVENT_PATH)
```

Comment:
Are `issuetitle` and `labels` used in this workflow?

Reply:
Nope! Just copied this block from a different workflow - removed 😄